elasticapm/traces.py

```diff
@@ -491,7 +516,7 @@ def end_transaction(self, result=None, transaction_name=None):
 
 
 class capture_span(object):
-    __slots__ = ("name", "type", "subtype", "action", "extra", "skip_frames", "leaf", "tags")
+    __slots__ = ("name", "type", "subtype", "action", "extra", "skip_frames", "leaf", "labels")
```

Comment:
don't you need both `tags` and `labels` here?

Reply:
no, the `capture_span` object doesn't have a `tags` attribute :)

Reply:
ah you are right ofc!